go/vt/orchestrator/logic/tablet_discovery.go

```diff
@@ -49,15 +49,21 @@ func OpenTabletDiscovery() <-chan time.Time {
 	if _, err := db.ExecOrchestrator("delete from vitess_tablet"); err != nil {
 		log.Errore(err)
 	}
-	if IsLeaderOrActive() {
-		RefreshTablets()
-	}
+	refreshTabletsUsing(func(instanceKey *inst.InstanceKey) {
+		_ = inst.InjectSeed(instanceKey)
+	})
 	// TODO(sougou): parameterize poll interval.
 	return time.Tick(15 * time.Second) //nolint SA1015: using time.Tick leaks the underlying ticker
 }
 
 // RefreshTablets reloads the tablets from topo.
 func RefreshTablets() {
```

Comment:
What's the purpose/usage of this function?

Reply:
This is called from the main discovery loop. It polls for new/deleted tablets, and updates Orc accordingly.

Reply:
got it. Makes sense; one thing to note is that this causes a synchronous topology read on all tablets. Possibly this isn't what you wanted? Consider calling `DiscoverInstance()` instead, which will queue the instance for discovery, and avoid double-probing it within a pre configured timeslice.